sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -113,18 +108,15 @@ foreach (ConversationEntity entity in conversationPrediction.Entities)
     Console.WriteLine($"Confidence: {entity.Confidence}");
     Console.WriteLine();
 
-    if (entity.Resolutions != null)
+    foreach (BaseResolution resolution in entity.Resolutions)
     {
-        foreach (BaseResolution resolution in entity.Resolutions)
+        if (resolution is DateTimeResolution)
         {
-            if (resolution is DateTimeResolution)
-            {
-                DateTimeResolution dateTimeResolution = resolution as DateTimeResolution;
-                Console.WriteLine($"Datetime Sub Kind: {dateTimeResolution.DateTimeSubKind}");
-                Console.WriteLine($"Timex: {dateTimeResolution.Timex}");
-                Console.WriteLine($"Value: {dateTimeResolution.Value}");
-                Console.WriteLine();
-            }
+            DateTimeResolution dateTimeResolution = resolution as DateTimeResolution;
```

Comment:
Nit: if `resolution` is not a `DateTimeResolution` it will be `null` and all the accesses below will throw a `NullReferenceException`. Instead, I'd use a cast here which will at least throw a more obvious `InvalidCastException`:

```csharp
DateTimeResolution dateTimeResolution = (DateTimeResolution)resolution;
```

Reply:
Actually, I missed in context line 113. You can do this all more simply and succinctly:

```csharp
if (resolution is DateTimeResolution dateTimeResolution)
{
  Console.WriteLine($"Datetime Sub Kind: {dateTimeResolution.DateTimeSubKind}");
  ...
}
```

Reply:
that's actually interesting.